src/main/java/de/tum/in/www1/artemis/domain/metis/conversation/Conversation.java

```diff
@@ -100,4 +101,24 @@ public Set<Post> getPosts() {
     public void setPosts(Set<Post> posts) {
         this.posts = posts;
     }
+
+    public String getHumanReadableName(User sender) {
+        if (this instanceof Channel channel) {
```

Comment:
You could override `getHumanReadableName(User sender)` in the subclasses of `Conversation` and you wouldn't need any `instanceof` checks. Is there any reason why you decided not to implement it like that?

Reply:
Good suggestion. I changed it!